lib/pengine/common.c

```diff
@@ -237,49 +237,42 @@ static pcmk__cluster_option_t pe_opts[] = {
             PCMK__VALUE_ONLY_GREEN ", " PCMK__VALUE_PROGRESSIVE ", "
             PCMK__VALUE_CUSTOM,
         PCMK__VALUE_NONE, pcmk__validate_health_strategy,
-        "How cluster should react to node health attributes",
-        "Requires external entities to create node attributes (named with "
-            "the prefix \"#health\") with values \"" PCMK__VALUE_RED "\", "
-            "\"" PCMK__VALUE_YELLOW "\", or \"" PCMK__VALUE_GREEN "\"."
+        N_("How cluster should react to node health attributes"),
+        N_("Requires external entities to create node attributes (named with "
+            "the prefix \"#health\") with values \" red \", "
+            "\" yellow \", or \" green \".")
```

Comment:
As an aside, I wish there were a way to avoid replacing the constants, since we lose the ability to catch typographical mistakes at compile-time, but I don't see a straightforward solution.

In the longer term, one possibility might be to avoid static initialization, change these struct members from `const char*` to `char*`, and create a function that adds one option at a time. All these strings would become the function arguments and could be dynamically generated as formatted strings (using `%s` where the defined constants are). Unfortunately, that would be a significant project, it wouldn't handle similar issues with GOption, and it would add the complexity of memory management.